README.md

```diff
@@ -98,635 +98,3 @@ You can select "luci-app-bmx7" to have a nice web interface for manage and monit
 
 Finally type "make" to build the image.
 
-## Usage (hello mesh)
```

Comment:
You need to review the ToC to link to the new files.

Reply:
Yes that's one of the TODOs concerning the README.

Reply:
So you'll update that?

Reply:
Yup